_posts/2024-05-20-highlights-otp-27.md

```diff
@@ -386,36 +401,41 @@ quotes() ->
     io:nl().
 ```
 
-Here follows a few quick examples to show the other sigils.
+Using `~B` would produce the exact same binary. Using the default
```

Comment:
Förklaringen här känns lite svår att förstå eftersom exemplet ovan inte faktiskt innehåller någon backslash som skulle kunna ändras av `~B` eller `~b` eller bara `~`. Kanske bäst att bara ta bort hela meningen "Using the default sigil ...", eller hitta på ett exempel med backslash som ska bevaras.